src/main.c

```diff
@@ -186,6 +186,7 @@ main (int argc, char *argv[])
   {
     buf_check_with_strop (undo_path, HOMEDIR, CPY);
     buf_check_with_strop (undo_path, UNDO_FILE, CAT);
+    int rmwed_files;
```

Comment:
You initialized this variable in your first draft. Let's do it again. Make it '0' ;)